apps/server/src/adapters/OscAdapter.ts

```diff
@@ -5,14 +5,15 @@ import { Server } from 'node-osc';
 import { IAdapter } from './IAdapter.js';
 import { dispatchFromAdapter } from '../controllers/integrationController.js';
 import { logger } from '../classes/Logger.js';
+import { error } from 'console';
```

Comment:
can you clarify this import?

Reply:
ah i didn't catch that 
no that is VS code auto importing stuff